Interfaces/API/Job.py

```diff
@@ -332,7 +329,7 @@ def setInputData( self, lfns ):
 
   #############################################################################
   def setParametricInputData( self, lfns ):
-    """Helper function.
+    """Helper function. Obsoleted, for backward compatibility only !
```

Comment:
Use the deprecated markup
http://www.sphinx-doc.org/en/stable/markup/para.html#directive-deprecated